tests/e2e/api-core-tests/data/shared/error-response.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * API error response format
+ *
+ * https://woocommerce.github.io/woocommerce-rest-api-docs/#errors
+ */
+const errorResponse = {
```

Comment:
Is it possible that there would be a list of error codes in the future? If so, let's start this as a multi-dimensional `errorResponses` object.

Reply:
This isn't being used anywhere yet so it's okay to leave it as a placeholder for this PR.

Reply:
Sounds good!